e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/add-to-an-existing-payment.spec.js

```diff
@@ -140,7 +158,13 @@ context(`${PDC_TEAMS.PDC_RECONCILE} users can add fee records to existing paymen
       pages.utilisationReportAddToAnExistingPaymentPage.continueButton().click();
 
       pages.utilisationReportPage.premiumPaymentsTab.getPaymentLink(PAYMENT_ID_ONE).should('contain', 'GBP 450.00');
-      pages.utilisationReportPage.premiumPaymentsTab.getPremiumPaymentsTableRow(FEE_RECORD_ID_ONE).should('contain', 'MATCH');
+
+      pages.utilisationReportPage.premiumPaymentsTab.premiumPaymentsTable
+        .status(FEE_RECORD_ID_ONE)
+        .invoke('text')
+        .then((text) => {
+          expect(text.trim()).to.equal('MATCH');
```

Comment:
Do we have constant for these like above, `FEE_RECORD_STATUS.DOES_NOT_MATCH`, or will this be part of future constants updates?

Reply:
Updated this instance and other instances in this PR, the others already in the codebase can be replaced under the new ticket